common/changes/@itwin/core-frontend/marc.bedard8-update_displayTestApp_2021-12-16-18-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Swap old @bentley/reality-data-client with new @itwin/relaity-data-client, Update DisplayTestApp and add two new utility function and update API",
```

Comment:
Doesn't look like this changelog is needed anymore

Reply:
Removed